cub/test/c2h/generators.cu

```diff
@@ -36,12 +36,31 @@
 
 #include <c2h/custom_type.cuh>
 #include <c2h/generators.cuh>
-#include <curand.h>
 #include <fill_striped.cuh>
 
+#if C2H_HAS_CURAND 
+#include <curand.h>
+#else
```

Comment:
No Change requested: We recently tried to be more strict with our conditional compilation. If you dont mind you can change it to `else // ^^^ C2H_HAS_CURAND ^^^ / vvv !C2H_HAS_CURAND  vvv`

It is an complete overkill here, so feel free to ignore

Reply:
Yeah, I'm trying not to introduce a text noise when it's not helpful. If it's a few lines, I'm omitting the annotation.